# Represents the state of an object

string object_name
string object_mesh_path # E.g, package://object_meshes/object_models/bowl_1k.obj
geometry_msgs/Pose object_pose # Pose of the object in the base frame of the robot
